utils_cv/tracking/plot.py

```diff
@@ -24,39 +22,24 @@ def plot_results(
         results: dictionary mapping frame id to a list of predicted TrackingBboxes
         input_video: path to the input video
         output_video: path to the output video
-        frame_rate: frame rate  
     """
-    # convert results to dataframe in MOT challenge format
-    preds = OrderedDict(sorted(results.items()))
-    bboxes = [
-        [
-            bb.frame_id,
-            bb.track_id,
-            bb.left,
-            bb.top,
-            bb.right - bb.left,
-            bb.bottom - bb.top,
-        ]
-        for _, v in preds.items()
-        for bb in v
-    ]
-
-    df = pd.DataFrame(
-        bboxes, columns=["frame", "id", "left", "top", "width", "height",],
+    _write_video(
+        OrderedDict(sorted(results.items())),
+        input_video,
+        output_video,
     )
 
-    _write_video(df, input_video, output_video, frame_rate)
-    print(f"Output saved to {output_path}.")
+    print(f"Output saved to {output_video}.")
 
-    return output_path
+    return output_video
 
 
 def _write_video(
-    df: pd.DataFrame, input_video: str, output_video: str
+    results: Dict[int, List[TrackingBbox]], input_video: str, output_video: str
```

Comment:
Should this function be called write_video() (without the "_") and be moved to model.py ?